packages/remix/src/generators/setup-tailwind/setup-tailwind.impl.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  addDependenciesToPackageJson,
+  formatFiles,
+  generateFiles,
+  installPackagesTask,
+  joinPathFragments,
+  readProjectConfiguration,
+  toJS,
+  type Tree,
+} from '@nx/devkit';
+
+import { upsertLinksFunction } from '../../utils/upsert-links-function';
+import { tailwindVersion } from '../../utils/versions';
+import { updateRemixConfig } from './lib';
+import type { SetupTailwindSchema } from './schema';
+
+export default async function setupTailwind(
+  tree: Tree,
+  options: SetupTailwindSchema
+) {
+  const project = readProjectConfiguration(tree, options.project);
+  if (project.projectType !== 'application') {
+    throw new Error(
+      `Project "${options.project}" is not an application. Please ensure the project is an application.`
+    );
+  }
+
+  updateRemixConfig(tree, project.root);
+
+  generateFiles(tree, joinPathFragments(__dirname, 'files'), project.root, {
+    tpl: '',
+  });
+
+  if (options.js) {
+    tree.rename(
+      joinPathFragments(project.root, 'app/root.js'),
```

Comment:
question: I've never used Remix without TypeScript (because I'm not a monster), but wouldn't the file be `root.jsx`?

Reply:
The `toJS` helper from Nx writes the files to `.js` rather than `.jsx`.